api/rpc/stat/stat.proto

```diff
@@ -3,14 +3,29 @@ syntax = "proto3";
 package stat;
 
 service Stat {
-  rpc ExecuteQuery (QueryRequest) returns (QueryReply) {}
+  rpc CPUQuery (CPURequest) returns (CPUReply) {}
 }
 
-message QueryRequest {
-  string database = 1;
-  string query=2;
+message CPURequest {
+  string ressourceName=1;
+  string filterDatacenter=2;
+  string filterServiceName=3;
+  string filterServiceID=4;
+  string filterContainerName=5;
+  string filterContainerID=6;
+  string filterContainerImage=7;
+  string filterNodeID=8;
 }
 
-message QueryReply {
-  string response = 1;
+message CPUEntry {
```

Comment:
I think there are two different measurements. docker and then system. The Docker stats are stored as int, but system is float